src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -498,13 +499,20 @@ private void buildBreathingSpaceEvent(EventHistory.EventHistoryBuilder builder,
     private void buildCcjEvent(EventHistory.EventHistoryBuilder builder, CaseData caseData) {
         if (caseData.isCcjRequestJudgmentByAdmission()) {
             buildJudgmentByAdmissionEventDetails(builder, caseData);
+
+            String miscTextRequested = RPA_REASON_JUDGMENT_BY_ADMISSION;
+            if (featureToggleService.isJOLiveFeedActive()
```

Comment:
I think we should not use state CaseState.All_FINAL_ORDERS_ISSUED as live feed can happen in any state and it should be consistant

Reply:
it's removed now